scripts/yocto-build-and-test.sh

```diff
@@ -145,9 +145,11 @@ EOF
         if [ -z "$mender_gateway_version" ]; then
             mender_gateway_version="master-git%"
         fi
+        local mender_gateway_examples_filename=$(find $WORKSPACE/stage-artifacts/ -maxdepth 1  -name "mender-gateway-examples-*.tar" | head -n1 | xargs basename)
```

Comment:
just to make me calm: it is `.tar` not `.tar.gz`, right?

Reply:
[Yes](https://github.com/mendersoftware/mender-gateway/blob/eb5e727f2c9ac9f84ced2894a0dd5d3aab68d656/Makefile#L100).